### ISSUE TITLE
Improve "Getting Started/Setup"

### ISSUE THREAD

**Issue by bkchr** [tien/reactive-dot#250]
When reading the docs I found the following:

> First add ReactiveDOT, along with required packages as dependencies to your React project.

For dumb developers like me, it would be really nice to at least link to some react tutorial or even better, to provide the commands to do this :)

**Comment by tien** [comment]
I must admit, the documentation can use a **lot** of improvements 😅

For this instance, I believe a separate tutorial makes sense, because there are like a bazillion different ways & commands involved in setting up a React application :p

Do you prefer an interactive tutorial, like [this](https://dotcodeschool.com/courses/intro-to-papi) for example? Or just something simple like a one-pager with all the basics?

**Comment by bkchr** [comment]
I'm personally happy with some commands to copy :D

**Comment by tien** [comment]
In that case, a "one command" to scaffold a basic DApp would be the way to go. I was told (sorry if this is not suppose to be public yet :p) by @mutantcornholio that Parity is working on that for PAPI, which may also includes support for ReactiveDOT.

**Comment by mutantcornholio** [comment]
Hi @bkchr!
You can check out this project https://github.com/paritytech/product-engineering/issues/104, perhaps I'm building the thing that you're looking for :)  
It'll support multiple templates, but the first template is based on ReactiveDOT.

**Comment by niklasp** [comment]
@mutantcornholio can you update the link or make it public? i am interested but 404

**Comment by mutantcornholio** [comment]
@niklasp Yeah, the link is Parity-only, but the project is in beta already, you can check it out here: https://github.com/paritytech/create-polkadot-dapp

I'm planning to announce it on Polkadot forum _probably_ later this week, after we merge [this PR](https://github.com/paritytech/create-polkadot-dapp/pull/8)
